environments/.stackhpc/terraform/main.tf

```diff
@@ -74,6 +78,7 @@ module "cluster" {
     cluster_image_id = data.openstack_images_image_v2.cluster.id
     control_node_flavor = var.control_node_flavor
     k3s_token = var.k3s_token
+    compute_init_enable = var.compute_init_enable
```

Comment:
Just set this directly here, there's no need for another variable/file. Although following other comments this will want to be defined it the compute= block.